labgrid/resource/udev.py

```diff
@@ -475,8 +475,8 @@ class YKUSHPowerPort(USBResource):
         serial (str): serial of the YKUSH device
         index (int): port index
     """
-    serial = attr.ib(default=None, validator=attr.validators.instance_of(str))
-    index = attr.ib(default=None, validator=attr.validators.instance_of(int),
+    serial = attr.ib(default="dummy", validator=attr.validators.instance_of(str))
+    index = attr.ib(default=-1, validator=attr.validators.instance_of(int),
```

Comment:
Adding suggest support this way only seems to work, because the serial is not checked during match. We should either have matching based on the path and autodetect the serial or the other way around, not require both to be specified manually. That would likely require some more intelligence in suggesst.

Reply:
Ok, I'll drop suggest support for now. We can add it later in a seperate pull request.